shared/constants/transaction.js

```diff
@@ -49,6 +50,7 @@ export const TRANSACTION_TYPES = {
   TOKEN_METHOD_APPROVE: 'approve',
   INCOMING: 'incoming',
   SENT_ETHER: 'sentEther',
+  SENDING_NATIVE_ASSET: 'simpleSend',
```

Comment:
Let's update the variable name too. So that it is descriptive of the actual transaction type.

Reply:
✅